src/test/java/org/folio/edge/sip2/api/support/BaseTest.java

```diff
@@ -85,6 +86,48 @@ void shutdown(Vertx vertx, TestInfo testInfo) {
     log.info("Finished: {}", testInfo.getDisplayName());
   }
 
+  /**
+   * Calls the service.
+   * @param ncipMessage the sip message to send.
```

Comment:
Just out of curiosity, why `ncipMessage` and not `sipMessage` here? I've seen that in some other test code and thought, "Hmmm...".

Reply:
typos :) I'll fixed them.